### ISSUE TITLE
Command names in English, with the Vietnamese name kept as an alias

### ISSUE THREAD

**Issue by tamnd** [tamnd/gao#147]
## The problem

Every subcommand of `gao` is a Vietnamese verb. That was a deliberate choice and it reads well in the source tree, where `phoi`, `sang` and `xay` sit next to each other as packages and the naming carries the metaphor of rice processing all the way through. It reads badly at a terminal. Somebody who has just cloned the repo and wants to know what runs the cleaning line has to read `gao help` end to end, and even then `sach` and `sang` and `siet` are three strings that look alike and do completely different things. The short descriptions carry the English gloss already, which is the tell: the name needs a translation to be usable.

So the command name should be the English verb, and the Vietnamese name should keep working as an alias. Nothing breaks, every script and every line in the README keeps running, and `gao clean` and `gao sach` are the same command.

## What changes

`command` grows an alias, the English name becomes the primary one, and the dispatch in `run` matches either. `usage` prints the English name with the Vietnamese one beside it, so the metaphor is still visible to anybody reading the help.

```go
type command struct {
	name  string // the English verb, which is what usage prints first
	alias string // the Vietnamese one, which is what the packages are named
	short string
	run   func(stdout, stderr io.Writer, args []string) int
}
```

The package names do not change. `phoi`, `sang`, `che` and `xay` stay as they are, because a package name is read by people working in the source and the source is where the metaphor belongs. This is about the surface a user types.

## The mapping

| English | Vietnamese | what it does |
| --- | --- | --- |
| board | bang | the release scores, native benchmarks kept apart from translated ones |
| frontier | bien | canonicalize URLs and shape the crawl frontier |
| husk | boc | peel the posts out of a forum thread |
| fleet | box | the fleet inventory and the disk budget it implies |
| weigh | can | whether the continued pretraining arms differ in their data and nothing else |
| grade | cham | grade sampled answers against a verifier |
| cover | che | find the personal data in a document and tag over it |
| route | chia | route a PDF to direct extraction, legacy transcode, or OCR |
| sink | chim | what an FP8 E4M3 step lost to zero |
| wait | cho | what the crawl left between requests to one host |
| choose | chon | score the base models against the six criteria |
| seal | chot | the evaluation harness, fixed and hashed before any result exists |
| total | cong | what a release holds, what of it ships, what the headline number counts |
| mark | dau | the diacritic restoration task set |
| count | dem | the tokenizer that defines a gao token, and what an ingest counted |
| fill | dien | the cloze proxy the ablation slate is run against |
| attach | dinh | page images kept joined to the text that came off them |
| predict | doan | the predictions register, written before the measurements |
| clear | don | whether the crawl gets its bytes off the box faster than it writes them |
| harvest | gat | the ingest manifest, and the check for drift |
| graft | ghep | what adding Vietnamese tokens to a base vocabulary bought and cost |
| stretch | gian | the context extension ladder |
| assign | giao | which box fetches which file of the ingest |
| sow | gieo | the generator card for gao-synth |
| keep | giu | what the distilled model kept of each specialist's gain |
| pack | goi | what a release costs on disk, column by column |
| efficiency | hieu | what fraction of the hardware a training run turns into gradient |
| ask | hoi | whether a question about a long document actually needs the document |
| pull | keo | what it costs to get back into a training run once the host is gone |
| store | kho | verify a snapshot, generate a signing key |
| needle | kim | whether a long context in Vietnamese is read or skimmed |
| repeat | lap | whether a generated set is a corpus or one prompt run a million times |
| slice | lat | check a release slice is a view over its snapshot rather than a second copy |
| law | luat | counsel questions, license determinations, what ships |
| seed | mam | find hosts nobody handed us a list of |
| sample | mau | which shards of a layer nobody has read get read |
| cook | nau | the training plan, its token budget, and the arithmetic between them |
| taste | nem | what a stored byte of each layer holds |
| hesitate | ngai | vi-overrefusal, whether a model will talk about Vietnam |
| listen | nghe | whether a transcript belongs to the audio it came off |
| pick | nhat | find the documents that hold a benchmark gao is judged on |
| throughput | nhip | what each pipeline stage runs at, with the box on every number |
| normalize | phoi | normalize Vietnamese text, or report what normalizing it would do |
| plan | ke | the build plan: slices, gates, and kill criteria |
| clean | sach | run the whole line over the raw corpus and publish what comes out |
| sift | sang | measure documents and say which of them are Vietnamese prose |
| tighten | siet | the GRPO step the specialists are trained with |
| compare | so | a human evaluation read back |
| inspect | soi | measure what a machine read against what the page says |
| yield | suat | the crawl's net yield, per target class |
| separate | tach | read a forum page as the thread it is |
| layers | tang | what an estimate taken bucket by bucket is worth |
| follow | theo | vi-adherence, whether the answer comes back in the language of the question |
| try | thu | the forty run ablation slate |
| syllable | tieng | what a syllable-atomic tokenizer would govern, and what it gives up |
| trust | tin | whether the cheap benchmark orders recipes the way the expensive one does |
| mix | tron | the finetuning set, with native origin kept a column rather than a note |
| estimate | uoc | what a sampled count is worth, as an interval and as a stopping rule |
| spike | vot | whether the loss spiked, and what rewinding would have cost |
| mill | xay | find the documents a corpus holds more than one copy of |
| place | xep | the gao-refset draw and rubric |
| takedown | xoa | who asked us to remove something, and how long it took |

`version` and `help` are already English and get no alias.

Two names are worth arguing about. `nhip` is the beat and `suat` is a rate, and both would translate to `rate`, so they go to `throughput` and `yield`, which is what each of them actually reports. `plan` was already English and had no Vietnamese name, so it gets `ke` for symmetry.

## Checklist

- [ ] `command` carries an alias, and `run` dispatches on either name
- [ ] `usage` prints the English name first with the Vietnamese one beside it
- [ ] every entry in the table above is wired, with the English name primary
- [ ] `gao help` output regenerated in the README
- [ ] a test that every command has a unique name, a unique alias, and no name that collides with another command's alias
- [ ] a test that every alias dispatches to the same function as its name
- [ ] the fleet scripts and the runbooks in the README updated to the English names
- [ ] run on a real box: `gao clean -plan` and `gao sach -plan` print the same thing

**Comment by tamnd** [comment]
Done in #149.

Every command answers to its English name and to the Vietnamese one. `gao clean` and `gao sach` are the same command, and nothing written against the old spelling stops working.

```
commands:
  ask         (hoi)    whether a question about a long document actually needs the document
  assign      (giao)   which box fetches which file of the ingest, and what the whole thing costs in wall clock
  ...
  clean       (sach)   run the whole line over the raw corpus and publish what comes out
  ...
  sift        (sang)   measure documents and say which of them are Vietnamese prose
```

Three things went further than the issue asked for, because leaving them would have made the rename half done.

Each subcommand had its own usage string and flag set named after the Vietnamese verb, so `gao clean -h` answered `usage: gao sach ...`. Those are renamed, 117 files of them.

The README's examples are rewritten to the English names, and the English glosses that used to sit in the trailing comments are gone. `gao mill parts/*.parquet # mill: what the corpus holds more than one copy of` now reads `gao mill parts/*.parquet # what the corpus holds more than one copy of`, because the name says it.

The Makefile and CI call three commands by name and those are renamed too, and all three were run against the real files rather than assumed.

Two things deliberately kept their old spelling. The predictions register keeps `gao kho reproduce` in the text of P06-3, because a claim written down before the measurement does not get reworded afterwards and the register is pinned by a digest for exactly that reason. The benchmark roster keeps `gao:kim` and `gao:ngai` as home addresses, because a roster address is data rather than documentation and the alias resolves it.

Four tests hold it: every name and alias is claimed by exactly one command, every verb has a Vietnamese name with `version` and `help` the only rows allowed to stand alone, both spellings of every command reach a command rather than the unknown command path, and the help prints both names for every row.

**Comment by tamnd** [comment]
Reopening the one decision this issue got wrong, for the record rather than to reargue it.

This issue said the package names do not change, because a package name is read by people working in the source and the source is where the metaphor belongs. In practice it meant translating twice, once from the command to the package and once from the package to what it does, and `gao clean` living in `sach/` is exactly the confusion the issue was filed about. #152 and #153 finish the rename: the packages, the files, and the identifiers are English now, and the Vietnamese verb stays where it explains itself, in the doc comment at the top of each package.
